### PR TITLE
Location vendor integration tests

### PR DESCRIPTION
**Pull request by jioter**
Location vendor integration tests

### CODE REVIEW

File: src/test/java/com/exadel/discountwebapp/location/service/LocationIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
+package com.exadel.discountwebapp.location.service;
+
+import com.exadel.discountwebapp.exception.EntityNotFoundException;
+import com.exadel.discountwebapp.location.entity.Location;
+import com.exadel.discountwebapp.location.repository.LocationRepository;
+import com.exadel.discountwebapp.location.vo.LocationRequestVO;
+import com.exadel.discountwebapp.location.vo.LocationResponseVO;
+import org.junit.Test;
```
Comment (ahlukhouski2) [inline src/test/java/com/exadel/discountwebapp/location/service/LocationIntegrationTest.java]: junit 5 should be used

File: src/test/java/com/exadel/discountwebapp/location/service/LocationIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
```
Comment (ahlukhouski2) [inline src/test/java/com/exadel/discountwebapp/location/service/LocationIntegrationTest.java]: @RunWith(SpringRunner.class) - is redundant for junit5 tests

File: pom.xml [hunk 3]
```diff
@@ -71,10 +72,12 @@
             <version>2.9.2</version>
         </dependency>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>5.4.2</version>
```
Comment (ahlukhouski2) [inline pom.xml]: redundant?

File: src/test/java/com/exadel/discountwebapp/location/service/LocationIntegrationTest.java (hunk 1 above)
Comment (jioter) [reply src/test/java/com/exadel/discountwebapp/location/service/LocationIntegrationTest.java]: Changed

File: src/test/java/com/exadel/discountwebapp/location/service/LocationIntegrationTest.java (hunk 2 above)
Comment (jioter) [reply src/test/java/com/exadel/discountwebapp/location/service/LocationIntegrationTest.java]: Removed

File: pom.xml (hunk 3 above)
Comment (jioter) [reply pom.xml]: Removed

**Review by dmitriychernov7 — APPROVED** [review]

File: src/test/java/com/exadel/discountwebapp/category/service/CategoryServiceIntegrationTest.java [hunk 4]
```diff
@@ -5,20 +5,17 @@
 import com.exadel.discountwebapp.category.vo.CategoryRequestVO;
 import com.exadel.discountwebapp.category.vo.CategoryResponseVO;
 import com.exadel.discountwebapp.exception.EntityNotFoundException;
-import org.junit.runner.RunWith;
+import org.assertj.core.util.Lists;
```
Comment (ahlukhouski2) [inline src/test/java/com/exadel/discountwebapp/category/service/CategoryServiceIntegrationTest.java]: ?
lets use guava or apache commons

Comment (jioter) [reply src/test/java/com/exadel/discountwebapp/category/service/CategoryServiceIntegrationTest.java]: Do you think it's necessary to add a new dependency for one-line change?
There is in-build method in org.assertj.core.util.Lists. 
It looks useful.

Comment (jioter) [reply src/test/java/com/exadel/discountwebapp/category/service/CategoryServiceIntegrationTest.java]: Changed to Guava

**Review by Bozhokmaria — APPROVED** [review]

**Pull request merged**